drivers/thunderbolt/README.md

```diff
@@ -66,3 +66,7 @@ You can also verify everything in dmesg:
  SUBSYSTEM=thunderbolt
  DEVICE=+thunderbolt:1-1
 ```
+
```

Comment:
I think an ideal solution would be to have a new talos machine config document for udev rules that gets applied even in maintenance, wdyt @smira